Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM node:16-alpine as tpb_custom_endpoint_service
```

Comment:
If we don't run the container in development, only dist should be copied into the container.

Reply:
sure